infrastructure/docker-registry/README.md

```diff
@@ -1,74 +1,88 @@
-```yaml
-ingress:
-  enabled: true
-  path: /
-  hosts:
-    - <registry.domain.name>
-  annotations:
-    cert-manager.io/cluster-issuer: letsencrypt-production
-  labels: {}
-  tls:
-    - hosts:
-      - <registry.domain.name>
+### Redis Configuration
+For the installation of Redis service, please see the link on the pre-requisites section about the `High available Redis`.
+In our architecture we have a [Redis-Sentinel](https://redis.io/topics/sentinel) service, instead of [Redis Cluster](https://redis.io/topics/cluster-tutorial), because with this architecture Sentinel manages automatically the failover of Master.
+It is possible have a `Redis-Sentinel ` architecture by setting the following parameter in the file values of redis_service_values.yaml file:
+  ```yaml 
+  sentinel.enabled=true
+  ```
+In the file you have to set an IP for the ClusterIP of the sentinel where after harbor is attached in the cluster and a name for your master. If you want have the replicas of redis running in different nodes you have to set the replica.podAntiAffinityPreset parameter to soft or hard, depending what you need. Please see this [link](https://docs.bitnami.com/tutorials/assign-pod-nodes-helm-affinity-rules/) to see the differences.
+To deploy Redis service, please see the link on the pre-requisites section or it is possible to proceed as follow:
+```bash
+$ helm repo add bitnami https://charts.bitnami.com/bitnami
+$ helm install my-release bitnami/redis
+$ helm upgrade --install redis bitnami/redis -f values.yaml
 ```
-
-  3. Configuration of the persistent disk. Please dimension this disk appropriately, as VM images may be rather large and hence you may fill up this space in a very short time.
-```yaml
-persistence:
-  accessMode: 'ReadWriteOnce'
-  enabled: true
-  size: 500Gi
-  storageClass: 'rook-ceph-block'
+the last commannd install/upgrade the chart with the release name `redis` and apply the configuration specified by the `values.yaml`.
+  
+### Postgres Configuration
+For the installation of Postgres service, please see the link on the pre-requisites section about the `High available PostgreSQL 9.6+` or it is possible to proceed as follow:
+```bash
+$ kubectl apply -f values.yaml
 ```
+This command create the database and apply the confguration specified by the `values.yaml`.
+
+### Harbor Configuration
+The following outlines the most relevant modifications applied to the Harbor `values.yaml` file:
+
+  1. Annotations concerning the configuration of the Ingress controller. You have to set the type of the ingress (in our file values.yaml the type of the ingress is ingress) in the expose.type parameter.
+
+  2. The external URL to expose the Harbor Registry. To set the external URL you have to set the externalURL parameter
+
+  3. For the external redis service you have to modify the redis.type parameter to external and have to set the same master name of the previous redis service in redis.external.sentinelMasterSet parameter.
```

Comment:
Configuration of the parameters to access the Redis service deployed previously.